core/services/job_services.py

```diff
@@ -111,6 +112,14 @@ def store_job_result(job_id, classifier_data):
     Returns:
         int. Status code of response.
     """
+    # The classifier data to be sent in the payload should have all
```

Comment:
Add link to https://stackoverflow.com/q/40173295

Reply:
Done.